release-tools/build.make

```diff
@@ -60,9 +60,27 @@ endif
 # Specific packages can be excluded from each of the tests below by setting the *_FILTER_CMD variables
 # to something like "| grep -v 'github.com/kubernetes-csi/project/pkg/foobar'". See usage below.
 
-build-%:
+BUILD_IN_DOCKER?=false
+DOCKER_BUILDER_WD=/go/src/github.com/kubernetes-csi/external-attacher
+DOCKER_BUILDER_IMG?=golang:$(shell grep go: $(CURDIR)/release-tools/travis.yml | cut -d: -f2,2 | tr -d ' ')
+
+vendor: Gopkg.toml Gopkg.lock
```

Comment:
The PR says that it is meant to enable *building* the source code in Docker, but here you make it possible to *vendor* under Docker, which is something that should only be necessary when editing or maintaining the source code.

In that case I think that developers won't get around installing the Go toolchain on their development system, for example to have access to "go fmt". "make test" also needs Go.

Reply:
... I'll remove then, no prob